main_test.go

```diff
@@ -72,6 +64,15 @@ func OpenTestConnection() (db *gorm.DB, err error) {
 		fmt.Println("testing sqlite3...")
 		db, err = gorm.Open("sqlite3", filepath.Join(os.TempDir(), "gorm.db"))
 	}
+
+	// db.SetLogger(Logger{log.New(os.Stdout, "\r\n", 0)})
+	// db.SetLogger(log.New(os.Stdout, "\r\n", 0))
+	if os.Getenv("DEBUG") == "true" {
+		db.LogMode(true)
+	}
+
+	db.DB().SetMaxIdleConns(10)
+
```

Comment:
Making sure the logging persists when connection is reopened in one of the tests.